source/MaterialXGenShader/UnitConverter.cpp

```diff
@@ -0,0 +1,208 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <algorithm>
```

Comment:
Minor:  This standard library should already be included through MaterialXCore/Library.h.